src/components/Modals/TourWindow.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <div class="tour">
+    <div class = "blackout">
+      <div class ="intropage">
+        <div class = "picture">
+          <img src = "@/assets/images/Person_planning.svg" alt = "person planning">
+        </div>
+        <div class = "content">
+          <div class = "title">
+            {{ title }}
+          </div>
+          <div class = "body" v-html = text>
+          </div>
+          <button v-on:click = "$emit('hide'); $emit('startTour')">
+            {{ buttonText }}
+          </button>
+          <a v-on:click = "$emit('skip')">
+            {{ exit }}
+          </a>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+const clickOutside = {
+  bind(el, binding, vnode) {
+    el.event = event => {
+      if (!(el === event.target || el.contains(event.target))) {
+        vnode.context[binding.expression](event, binding.arg);
+      }
+    };
+    document.body.addEventListener('click', el.event);
+  },
+  unbind(el) {
+    document.body.removeEventListener('click', el.event);
+  }
+};
+
+export default {
+  props: {
+    title: String,
+    text: String,
+    exit: String,
+    buttonText: String
+  },
+  data() {
+    return {
+      hideOnClick: ''
```

Comment:
I'm not exactly understanding why hideOnClick is a string? Could this be variable named a little better or am I not understanding something here?